### PR TITLE
Handle VAConfigAttribRTFormat query to AV1 profiles case explicitly

### PR DESCRIPTION
**Pull request by rosetta-jpn**
Signed-off-by: Hirokazu Honda <hiroh@chromium.org>

### CODE REVIEW

**Comment by rosetta-jpn** [comment]
I am not sure that gen12 supports 4:0:0, 10bpp and 12bpp.
Could anyone tell me?

**Comment by XinfengZhang** [comment]
as I know, media driver does not support mono chrome @ZefuLI please help to check it.

**Comment by rosetta-jpn** [comment]
How about 10bpp and 12bpp?

**Comment by rosetta-jpn** [comment]
Add 10bits. I suppose 4:4:4 is not supported on gen12.

**Comment by rosetta-jpn** [comment]
I hope this is ready for submitting.
Could anyone review this patch and check in?

Thanks in advance.

**Comment by rosetta-jpn** [comment]
gentle ping!

**Comment by rosetta-jpn** [comment]
@sreerenjb could you prioritize to check in this patch? This is required to play HDR (10bit) stream. (How have intel done 10bits stream test without this patch..?)

File: media_driver/linux/gen12/ddi/media_libva_caps_g12.cpp [hunk 1]
```diff
@@ -2009,6 +2009,10 @@ VAStatus MediaLibvaCapsG12::CreateDecAttributes(
             | VA_RT_FORMAT_YUV422_12
             | VA_RT_FORMAT_YUV444_12;
     }
+    else if (profile == VAProfileAV1Profile0 || profile == VAProfileAV1Profile1)
```
Comment (sreerenjb) [inline media_driver/linux/gen12/ddi/media_libva_caps_g12.cpp:2012]: AV1 profile1 doesn't have 420 support, it is exclusively 4:4:4.

**Comment by sreerenjb** [comment]
It seems there are more issues with av1 handling.
f (MEDIA_IS_SKU(&(m_mediaCtx->SkuTable), FtrIntelAV1VLDDecoding10bit420))
    {
        status = CreateDecAttributes((VAProfile) VAProfileAV1Profile1, VAEntrypointVLD, &attributeList);

@XinfengZhang   @ZefuLI

**Comment by XinfengZhang** [comment]
hi @rosetta-jpn , this PR is LGTM. but just like our discussion, we plan to remove profile1 support.  suppose @ZefuLI will file another pull request of AV1 profile1 removal.  we could merge it now, but the profile1 maybe will be removed in near future.

**Comment by rosetta-jpn** [comment]
Hi @sreerenjb and @XinfengZhang 
That sounds good. Thanks for verifying.

**Pull request merged**